tests/cockpit-tests

```diff
@@ -31,7 +33,7 @@ if ! whoami && [ -w /etc/passwd ]; then
 fi
 
 if [ ! -d cockpit ]; then
-    git clone https://github.com/cockpit-project/cockpit
+    git clone "$GITHUB_REPO"
```

Comment:
Since the project name is now encoded in `$GITHUB_REPO`, please change this to `git clone "$GITHUB_REPO" cockpit`. (Same further down.)

Reply:
Fixed.